api-lib/sendSocialMessage.test.ts

```diff
@@ -0,0 +1,102 @@
+import * as nodeFetch from 'node-fetch';
+
+import { createCircle } from '../api-test/helpers';
+
+import { adminClient } from './gql/adminClient';
+import { sendSocialMessage } from './sendSocialMessage';
+import { Awaited } from './ts4.5shim';
+
+const fetchOrig = jest.requireActual('node-fetch').default;
+
+const expectedResponse = {
+  ok: false,
+  status: 404,
+  statusText: 'Not Found',
+  headers: new Headers(),
+  text: jest.fn().mockResolvedValue(''),
+  json: () =>
+    Promise.resolve({
+      success: false,
+      error: 'Unexpected Id',
+    }),
+} as unknown as nodeFetch.Response;
+
+describe('sendSocialMessage', () => {
```

Comment:
can you add a test that sends two, has one fail, and verifies that the other one succeeds?

also, no need to have a `describe` block around all of this if there's only one in the whole test file